modules/audit-enqueued-assets/load.php

```diff
@@ -0,0 +1,312 @@
+<?php
+/**
+ * Module Name: Audit Enqueued Assets
+ * Description: Adds a CSS and JS resource checker in Site Health checks.
+ * Focus: site-health
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Audit enqueued scripts in the frontend. Ignore /wp-includes scripts.
+ *
+ * It will save information in a transient for 12 hours.
+ *
+ * @since 1.0.0
+ */
+function perflab_aea_audit_enqueued_scripts() {
+	if ( ! is_admin() && ! get_transient( 'aea_enqueued_scripts' ) ) {
+		global $wp_scripts;
+		$enqueued_scripts = array();
+
+		foreach ( $wp_scripts->queue as $handle ) {
+			$src = $wp_scripts->registered[ $handle ]->src;
+			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+				$enqueued_scripts[] = array(
+					'src'  => $src,
+					'size' => perflab_get_resource_file_size( perflab_get_path_from_resource_url( $src ) ),
+				);
+			}
+		}
+		$expiration = apply_filters( 'perflab_aea_audit_enqueued_scripts_expiration_in_seconds', 12 * HOUR_IN_SECONDS );
```

Comment:
Do we need a filter here? Not opposed if there's a good reason for it, but I think we should only introduce it if we have a good use-case for it, and keep things simple where we can.

Reply:
@felixarntz this was introduced in case some webmaster wants to control programmatically the expiration of this cache, in case of experiments or whatever the need may arise for them. Would it have a real use case? hard to know, but I think it doesn't hurt.

Reply:
Followed up in https://github.com/WordPress/performance/pull/54#discussion_r791704370

I still prefer to remove it since we don't have a good use-case in mind for it. But the other comment has some more nuance, let's follow up there.